server/src/state.rs

```diff
@@ -19,7 +19,7 @@ pub enum Slot<T: Clone> {
 
 pub struct NodesState {
     /// Constant time access using slot id optimized for forwarding.
-    /// The consequence is, that we must store Option<NodeSession>, because
+    /// The consequence is, that we must store `Option<NodeSession>`, because
```

Comment:
Wasn't `NodeSession` previously stored in an `Option<NodeSession>` and this comment is a remainder from back then?